public/js/main.js

```diff
@@ -0,0 +1,45 @@
+//Animating chat bubbles on conversation cards:
+
+window.onload = function() {
```

Comment:
Since your scripts are at the bottom of the page you _probably_ don't need this.